ptsprojects/stack.py

```diff
@@ -317,6 +317,17 @@ def __init__(self, uuid, oob, output_size, output_actions, input_size,
         # Node Identity
         self.proxy_identity = False
 
+        # Config Client
+        self.net_idx = 0x0000
+        self.addr = 0x0002
```

Comment:
This address will be dependent on whether IUT is a Provisioner or a Node. We need a mechanism to update it based on the role.